File: PRAK503-2310817210001-AndraBraputraAkbarSaleh.py
def maksimal(a, b):
    return a if a > b else b

def minimal(a, b):
    return a if a < b else b

def main():
    bilangan = int(input())
    nilai = list(map(int, input().split()))

    maks = nilai[0]
    minim = nilai[0]

    for hasil in nilai[1:]:
        maks = maksimal(maks, hasil)
        minim = minimal(minim, hasil)

    print(f"{maks} {minim}")

if __name__ == "__main__":
    main()

File: PRAK503-2310817210001-AndraBraputraAkbarSaleh.c
#include <stdio.h>
int maksimal(int a, int b){
    int a2 = (a > b) ? a : b;
    return a2;
}
int minimal(int a, int b){
    int b2 = (a < b) ? a : b;
    return b2;
}
int main(){
 int batas = 0;
 int maks = -100000;
 int minim = 100000;
 int bilangan;
 scanf("%d", &bilangan);
 while(batas < bilangan){
 int nilai;
 scanf("%d", &nilai); 
 maks = maksimal(maks, nilai);
 minim = minimal(minim, nilai);
 batas++;
 }
 printf("%d %d",maks,minim); 
}
